frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -5,102 +5,138 @@ import classnames from 'classnames'
 import { translate } from 'react-i18next'
 import PropTypes from 'prop-types'
 import AnimateHeight from 'react-animate-height'
+import { DropTarget } from 'react-dnd'
+import {DRAG_AND_DROP, ROLE_OBJECT} from '../../helper.js'
 
 const qs = require('query-string')
 
-const shouldDisplayAsActive = (location, idWorkspace, activeIdWorkspace, app) => {
-  if (idWorkspace !== activeIdWorkspace) return false
+class WorkspaceListItem extends React.Component {
```

Comment:
I have changed this stateless component to a statefull component.
That's why there is a big diff on it